src/database/migrations/20241212202720-add_role_extensions_data.js

```diff
@@ -0,0 +1,90 @@
+'use strict'
+
+require('module-alias/register')
+const common = require('@constants/common')
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		try {
+			// Data to be inserted
+			const roleExtensionsData = [
+				{
+					title: 'user',
```

Comment:
@sumanvpacewisdom do we have this role in user service ?

Reply:
@rakeshSgr 
yes we have in users services , we are using it in the bulkuserupload